### DISCUSSION TITLE
Kalevala to Stellarium

### DISCUSSION THREAD

**Discussion by ilkant** [Stellarium/stellarium#3722] [General]
In Finland there is a national epic Kalevala. Ursa is national association of astronomy in Finland. Their magazine Tähdet ja avaruus 3/2024 contained an article of Kalevala celestial. In year 1882 Edvard Neovius published his story and pictures of Kalevala epic to sky culture.

I tried to add that to Stellarium. I copied modern sky culture to new kalevala directory. Maybe pictures have no copyright anymore. But I have not found how to change constellations names. And how each stars are defined to build constellation.

**Comment by alex-w** [comment]
stellarium user guide has chapter for sky cultures
